demo/README.md

```diff
@@ -17,6 +17,7 @@ export WAGTAIL_NEWSLETTER_MAILCHIMP_API_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-us1
 export WAGTAIL_NEWSLETTER_FROM_NAME="My Newsletter"
 export WAGTAIL_NEWSLETTER_REPLY_TO=sender@example.com
 
+python -m pip install --editable='.[testing,mailchimp,mrml]'
```

Comment:
Just realised we do have a `CONTRIBUTING.md`... But OK, I think this is a small worthwhile improvement 🙂